lib/license_finder/package_managers/go_modules.rb

```diff
@@ -58,6 +58,7 @@ def packages_info
         go_list_cmd = "GO111MODULE=on go list -mod=readonly -deps -f '#{format_str}' ./..."
         info_output, stderr, status = Cmd.run(go_list_cmd)
```

Comment:
Would it be beneficial to also show the info_output (stdout) when it fails?

Reply:
Because the go command errors, `info_output` actually has no value.